osu.Game/Rulesets/Mods/ModFailCondition.cs

```diff
@@ -19,12 +19,31 @@ public abstract class ModFailCondition : Mod, IApplicableToHealthProcessor, IApp
         public virtual bool PerformFail() => true;
 
         public virtual bool RestartOnFail => Restart.Value;
+        private HealthProcessor healthProcessorInternal;
 
         public void ApplyToHealthProcessor(HealthProcessor healthProcessor)
         {
+            healthProcessorInternal = healthProcessor;
             healthProcessor.FailConditions += FailCondition;
         }
 
+        /// <summary>
+        /// Immediately triggers a failure on the loaded <see cref="HealthProcessor"/>.
+        /// </summary>
+        protected void TriggerArbitraryFailure() => healthProcessorInternal.TriggerFailure();
```

Comment:
Arguably no need for the "arbitrary" infix. Just `TriggerFailure()` would do imo.

Reply:
I added the arbitrary to distinguish it further from `PerformFail`, but I guess the "trigger" and included xmldoc is enough. Changed.